tests/file/ActiveRecordTest.php

```diff
@@ -303,7 +303,7 @@ public function testWriteFile()
     /**
      * @depends testInsertFileContent
      */
-    public function testGetFileResource()
+    /*public function testGetFileResource()
```

Comment:
Not now: need to consider if `Download::getResource()` is implementable.
It if does test can be re-enabled, otherwise - it will be removed.